apps/aeprimop/src/aeprimop.erl

```diff
@@ -341,9 +343,7 @@ channel_create_tx_instructions(InitiatorPubkey, InitiatorAmount,
                         ResponderPubkey, ResponderAmount,
                         ReserveAmount, DelegatePubkeys,
                         StateHash, LockPeriod, Nonce, Round)
-    , tx_event_op({channel, aesc_channels:pubkey(InitiatorPubkey,
-                                                 Nonce,
-                                                 ResponderPubkey)})
+    , tx_event_op({channel, ChannelPubkey})
```

Comment:
This is made GA aware. For channel create transactions wrappend in a meta authentication transaction(s) the channel pubkey is computed in a different GA-specific way (the nonce is 0, so we use the `auth_id` instead). This was emitting events with the wrong channel ID.

This is not breaking consesus but rather modifying the emitted events.

Reply:
This solves: [Check event emited when GA channel is created](https://www.pivotaltracker.com/story/show/166272180)